docs/automations/ai-docs.md

```diff
@@ -28,6 +28,18 @@ All AI-generated documentation that has not been reviewed by a human will be mar
   <img width="70%" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/automation/saas/ai-docs/ai-docs-generation.gif"/>
 </p>
 
+### Customize Documentation Generation
+
+You can customize how documentation is generated by providing custom instructions that are passed to the underlying AI model when generating dcumentation for any Table or Column. This is useful if you want AI-generated documentation to follow specific guidelines or standards set by your organization.
```

Comment:
Since we call out v0.3.12 above for feature avialiblty, let's be explicit in this scetion that's it's avail as of v0.3.15